code/go/0chain.net/smartcontract/dbs/event/snapshot.go

```diff
@@ -0,0 +1,59 @@
+package event
+
+import (
+	"errors"
+	"sync"
+
+	"0chain.net/core/logging"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+)
+
+// swagger:model Block
+type Snapshot struct {
+	gorm.Model
```

Comment:
not sure we need model here, it will create couple of columns and index for deletes that we don't use